### DISCUSSION TITLE
search_items "resources" keyword (python)

### DISCUSSION THREAD

**Discussion by dubidubi1** [sergioteula/python-amazon-paapi#127] [Help]
I'm not sure how to specify the "resources" parameter in search_items (the default response doesn't return ItemInfo.ContentInfo...at least it's None every time for me). Here's my request:
`search_items(browse_node_id='6459737011', condition='New', resources="ItemInfo.ContentInfo")`

I get the following error in response:
`MalformedRequest: Parameters for search_items request are not correct: amazon_paapi.sdk.models.search_items_request.SearchItemsRequest() got multiple values for keyword argument 'resources'`

The Traceback points me at the `requests.py` file, which references `SearchItemsResource`. The value I show above (`"ItemInfo.ContentInfo"`) IS in the list of valid values.

Help - how do I properly formulate this request?

**Comment by inbrighttech** [comment]
Late response but:
It seems the requests.py helper already embeds the resources variable with all the available resources for each of the operations.
The response should/might include the ItemInfo.ContentInfo.
